src/pages/apps/index.tsx

```diff
@@ -21,8 +21,8 @@ import { getOrigin } from '@/components/safe-apps/utils'
 const SafeApps: NextPage = () => {
   const chainId = useChainId()
   const router = useRouter()
-
-  const [appUrl, routerReady] = useSafeAppUrl()
```

Comment:
It seems like [we shouldn't have been using `router.isReady` for conditional rendering](https://nextjs.org/docs/api-reference/next/router#router-object) at all.